zap/src/main/java/org/zaproxy/zap/view/SplashScreen.java

```diff
@@ -75,8 +75,10 @@
 
     public SplashScreen() {
         super();
-
+        this.setUndecorated(true);
+        this.setAlwaysOnTop(true);
         setSize(DisplayUtils.getScaledDimension(420, 430));
+        setResizable(false);
```

Comment:
This is more a workaround, the progress bar and background image can have a fixed size.

Reply:
Does the splash screen really need to be resizable?

Reply:
No, but it's better if it can be (e.g. easier to read the tips and log when maximised).